demo/demo_sat.py

```diff
@@ -74,7 +79,27 @@ def __init__(self, name="Space Oddity"):
             }
         }
 
-    async def command_filter(self, command, major_tom):
+    async def cancel_callback(self, id, major_tom):
+        if str(id) in self.running_commands:
+            self.running_commands[str(id)]["cancel"] = True
+        else:
+            asyncio.ensure_future(major_tom.transmit_events(events=[{
```

Comment:
Won't the command now sit on MT forever in a hanging state?

Reply:
I figured that's better than faking it. It's possible that the cancel message could get received after the command fully completed if there is a massive queue, so I don't want to accidentally cause an incorrect state. 

If we want to clear commands out of Major Tom that are hanging like that, it might be a good to add a "force cancel button or something like that.